packages/wxt/src/builtin-modules/unimport.ts

```diff
@@ -1,29 +1,28 @@
 import { addViteConfig, defineWxtModule } from '../modules';
 import type {
   EslintGlobalsPropValue,
+  Wxt,
   WxtDirFileEntry,
   WxtModule,
   WxtResolvedUnimportOptions,
 } from '../types';
-import { type Unimport, createUnimport } from 'unimport';
+import { type Unimport, createUnimport, toExports } from 'unimport';
 import { Plugin } from 'vite';
 import { extname } from 'node:path';
 
 export default defineWxtModule({
   name: 'wxt:built-in:unimport',
   setup(wxt) {
-    const options = wxt.config.imports;
-    if (options === false) return;
```

Comment:
Now, instead of disabling the entire module, we only set a flag to false in the vite plugin to prevent adding auto-imports, but still replace `#imports`.